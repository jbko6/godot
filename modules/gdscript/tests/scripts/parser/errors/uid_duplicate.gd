@uid("uid://c4ckv3ryprcn4")
@uid("uid://c4ckv3ryprcn4")

func test():
    pass
