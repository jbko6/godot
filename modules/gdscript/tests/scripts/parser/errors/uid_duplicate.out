GDTEST_PARSER_ERROR
"@uid" annotation can only be used once.
